### DISCUSSION TITLE
Route grouping enhancement

### DISCUSSION THREAD

**Discussion by bostjanpisler** [wakujs/waku#1049] [Ideas]
Allow creating route groups using parenthesis in folder names, i.e. /pages/(marketing) inspired by Next.js.

This would allow creating multiple root routes with different layouts, i.e. /pages/(marketing)/_layout.tsx and /pages/(shop)/_layout.tsx

Example structure:
```
pages
    (marketing)
        _layout.tsx
        home.tsx
    (shop)
        _layout.tsx
        products.tsx
```
 
Example endpoints:
`/home`
`/products`

**Comment by dai-shi** [comment]
#977 is proposed, and we will be working on it later.
Does it look something similar?

**Reply by bostjanpisler** [reply]
Yes, I see it mentioned in the comments there, will close, thanks.
